packages/SwingSet/test/vat-admin/terminate/test-terminate.js

```diff
@@ -45,12 +45,41 @@ test('terminate', async t => {
     'query2 2',
     'QUERY 3',
     'GOT QUERY 3',
-    'foreverP.catch vat terminated',
-    'query3P.catch vat terminated',
-    'foo4P.catch vat terminated',
-    'afterForeverP.catch vat terminated',
+    'foreverP.catch Error: vat terminated',
+    'query3P.catch Error: vat terminated',
+    'foo4P.catch Error: vat terminated',
+    'afterForeverP.catch Error: vat terminated',
+    reference,
     'done',
   ]);
+}
+
+test('terminate', async t => {
+  await doTerminate(t, 'kill', 'done exception kill');
+});
+
+test('exit happy path simple result', async t => {
+  await doTerminate(t, 'happy', 'done result happy');
+});
+
+test('exit happy path complex result', async t => {
```

Comment:
since we're allowing `exitVat` to accept arbitrary serializable objects, we ought to have a test that demonstrates passing object (Presence) references too.. probably a new test file

Reply:
That was a good call.  Turns out the vat power implementations needed to harden the arg.  But it works now!